docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -221,7 +221,24 @@ Custom config is not yet supported for RocksDB.
 
 #### `--genesis` (string):
 
-Path to a JSON file containing the genesis data to use. Ignored when running standard networks (Mainnet, Fuji Testnet), or when `--genesis-content` is specified. If not given, uses default genesis data. For an example of a JSON representation of genesis data, see [here](https://github.com/ava-labs/avalanchego/blob/master/genesis/genesis_local.go#L16).
+Path to a JSON file containing the genesis data to use. Ignored when running standard networks (Mainnet, Fuji Testnet), or when `--genesis-content` is specified. If not given, uses default genesis data.
+
+This are the main properties in the JSON file:
+
+- `networkID`: A unique identifier for the blockchain, must be a number in the range [0, 2^32].
+- `allocations`: The list of initial addresses, their initial balances and the unlock schedule for each.
+- `startTime`: The time of the beginning of the blockchain, it must be a timestamp in seconds and it can't be a time in the future.
+- `initialStakeDuration`: The initial stake duration in seconds.
+- `initialStakeDurationOffset`: The initial stake duration offset between stakers in seconds.
+``` go
```

Comment:
[MD031 - Fenced code blocks should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md031)